src/deployments/kubernetes/app-configmap.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
     name: app-configmap
 data:
     postgres-user: postgres
+    postgres-admin-user: postgres
     postgres-host: postgres-service
-    postgres-port: "5432"
+    postgres-port: "5402"
     postgres-dbname: postgres
+    need-db-setup: "true"
```

Comment:
Please also update the example secrets file, adding the admin password key there.